app/components/concerns/katalyst/tables/orderable.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module Katalyst
+  module Tables
+    # Adds drag and drop ordering to a table.
+    # See [documentation](/docs/orderable.md) for more details.
+    module Orderable
+      extend ActiveSupport::Concern
+
+      FORM_CONTROLLER = "tables--orderable--form"
+      ITEM_CONTROLLER = "tables--orderable--item"
+      LIST_CONTROLLER = "tables--orderable--list"
+
+      using HasHtmlAttributes
+
+      def self.extended(table)
+        table.class.config_component :orderable, default: FormComponent
+      end
+
+      def orderable?
+        @orderable.present?
+      end
+
+      def orderable
+        @orderable
+      end
+
+      def with_orderable(url:,
+                         id: "#{self.id}-orderable-form",
+                         scope: "order[#{collection.items.model_name.plural}]")
+        header_row_component.include(HeaderRow)
+        body_row_component.include(BodyRow)
+        @orderable = FormComponent.new(table: self, url: url, id: id, scope: scope)
+      end
+
+      def tbody_attributes
+        super.merge_html(
+          { data: { controller: LIST_CONTROLLER,
+                    action: <<~ACTIONS.squish,
+                      dragstart->#{LIST_CONTROLLER}#dragstart
+                      dragenter->#{LIST_CONTROLLER}#dragenter
+                      dragover->#{LIST_CONTROLLER}#dragover
+                      drop->#{LIST_CONTROLLER}#drop
+                    ACTIONS
+                    "#{LIST_CONTROLLER}-#{FORM_CONTROLLER}-outlet" => "##{orderable.id}",
+                    "#{LIST_CONTROLLER}-#{ITEM_CONTROLLER}-outlet" => "td.ordinal" } }
+        )
+      end
+
+      module HeaderRow # :nodoc:
+        def ordinal(attribute = :ordinal, **)
+          cell(attribute, class: "ordinal", label: "")
+        end
+      end
+
+      module BodyRow # :nodoc:
+        def ordinal(attribute = :ordinal, id: :id)
+          name  = @table.orderable.record_scope(@record, id, attribute)
+          value = @record.public_send(attribute)
+          cell(attribute, class: "ordinal", data: {
+                 controller: ITEM_CONTROLLER,
+                 "#{ITEM_CONTROLLER}-name-value" => name,
+                 "#{ITEM_CONTROLLER}-value-value" => value
+               }) { t("katalyst.tables.orderable.value") }
+        end
+
+        def html_attributes
+          super.merge_html(
+            {
+              draggable: "true"
+            }
+          )
+        end
+      end
+
+      class FormComponent < ViewComponent::Base # :nodoc:
+        attr_reader :id, :url, :scope
+
+        def initialize(table:, id:, url:, scope:)
+          super
+
+          @table = table
+          @id = id
+          @url = url
+          @scope = scope
```

Comment:
Has our alignment cop changed? Seeing quite a few instances of this.

Reply:
Yeah I'm really confused why it's not doing anything

Reply:
Fixed 👍